api/proto/kent/api/worktree/worktree.proto

```
@@ -14,9 +14,10 @@ option go_package = "core/shared/protoapi/gen/kent/api/worktree;worktreepb";
 
 enum TopologyVariant {
   WORKTREE_TOPOLOGY_VARIANT_UNSPECIFIED = 0;
-  WORKTREE_TOPOLOGY_VARIANT_REGISTERED = 1;
-  WORKTREE_TOPOLOGY_VARIANT_EXTERNAL = 2;
-  WORKTREE_TOPOLOGY_VARIANT_MISSING = 3;
+  WORKTREE_TOPOLOGY_VARIANT_MAIN_WORKSPACE = 1;
```

Buf (1.72.0)

[error] Enum value "1" on enum "TopologyVariant" changed name from "WORKTREE_TOPOLOGY_VARIANT_REGISTERED" to "WORKTREE_TOPOLOGY_VARIANT_MAIN_WORKSPACE".

(ENUM_VALUE_SAME_NAME)